--- gtkextra-auto.f90.raw	2011-06-22 23:34:35.000000000 +0200
+++ gtkextra-auto.f90	2011-06-22 23:46:39.816981398 +0200
@@ -555,12 +555,12 @@
 
 !  void gtk_plot_data_set_points (GtkPlotData *data, gdouble *x, gdouble *y, gdouble *dx, gdouble *dy, gint num_points);
 subroutine gtk_plot_data_set_points(data, x, y, dx, dy, num_points) bind(c) 
-  use iso_c_binding, only: c_ptr, c_int
+  use iso_c_binding, only: c_ptr, c_int, c_double
   type(c_ptr), value :: data
-  type(c_ptr), value :: x
-  type(c_ptr), value :: y
-  type(c_ptr), value :: dx
-  type(c_ptr), value :: dy
+  real(c_double), dimension(*) :: x
+  real(c_double), dimension(*) :: y
+  real(c_double), dimension(*) :: dx
+  real(c_double), dimension(*) :: dy
   integer(c_int), value :: num_points
 end subroutine
 
